askama/src/filters/humansize.rs

```diff
@@ -23,12 +21,12 @@ use crate::{FastWritable, NO_VALUES, Values};
 /// assert_eq!(tmpl.to_string(),  "Filesize: 1.23 MB.");
 /// ```
 #[inline]
-pub fn filesizeformat(b: f32) -> Result<FilesizeFormatFilter, Infallible> {
-    Ok(FilesizeFormatFilter(b))
+pub fn filesizeformat(b: u128, p: u8) -> Result<FilesizeFormatFilter, Infallible> {
+    Ok(FilesizeFormatFilter(b, p))
 }
 
 #[derive(Debug, Clone, Copy)]
-pub struct FilesizeFormatFilter(f32);
+pub struct FilesizeFormatFilter(u128, u8);
```

Comment:
Could you use fields instead? Having context on what the types are actually for is always a good idea. :)

Reply:
This comment has not been resolved. :wink:

Reply:
Ohh sorry, I didn't look properly. I somehow thought you meant the SiPrefixes. Oh well...